nidx/nidx_types/src/lib.rs

```diff
@@ -30,6 +30,16 @@ impl From<u64> for Seq {
         Self(value as i64)
     }
 }
+impl From<i32> for Seq {
+    fn from(value: i32) -> Self {
+        Self(value as i64)
+    }
+}
+impl From<Seq> for i64 {
```

Comment:
This is fine, although it can be easily be avoided by doing `i64::from(&seq)` instead of `i64::from(seq)`.

Reply:
I think I prefer to keep both in this case if you don't mind